scan/connectivity_scanner.go

```diff
@@ -0,0 +1,57 @@
+package scan
+
+import "net"
+
+// ConnectivityScans contains scanners testing basic connectivity to the host
+var ConnectivityScanners = &Family{
+	Name:        "ConnectivityScanners",
```

Comment:
Style note: if Family denotes a family of scanners, you can probably drop the "Scanners" part of the name. if this is the name that the user selects the family with on the commandline, I would just use "connectivity" to make it easier and consistent with the other args.